samples/hosting/docker/Core_7/Receiver/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using NServiceBus;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Exceptions;
+
+class Program
+{
+    static async Task Main()
+    {
+        //required to prevent possible occurrence of .NET Core issue https://github.com/dotnet/coreclr/issues/12668
+        Thread.CurrentThread.CurrentUICulture = new CultureInfo("en-US");
+        Thread.CurrentThread.CurrentCulture = new CultureInfo("en-US");
+
+        Console.CancelKeyPress += new ConsoleCancelEventHandler(OnExit);
```

Comment:
Would it be easier to follow if this was like this?

```
Console.CancelKeyPress += (sender, args) => closingEvent.Set();
```

(In the other endpoint too.)